packages/table/src/table.tsx

```diff
@@ -826,6 +830,33 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         return this.maybeRenderRegions(styler);
     }
 
+    private maybeRenderMenuRegions() {
+        const styler = (region: IRegion): React.CSSProperties => {
+            const { grid } = this;
+            const { viewportRect } = this.state;
+            if (viewportRect == null) {
+                return {};
+            }
+            const cardinality = Regions.getRegionCardinality(region);
+            const style = grid.getRegionStyle(region);
+
+            switch (cardinality) {
+                case RegionCardinality.FULL_TABLE:
+                    style.right = "0px";
+                    style.bottom = "0px";
+                    style.top = "0px";
+                    style.left = "0px";
```

Comment:
🤔 hmmm pretty sure this entire file could use this change because React will add `"px"` to `number` values in CSS. 

so maybe it's not worth making here just yet. this is consistent, to be sure.